FightPandemics/Features/Feed/OfferRequestCard/Subheader-OfferReqCard.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Subheader-OfferReqCard.swift
+//  FightPandemics
+//
+//  Created by Luciano Schillagi on 6/15/20.
+//
+//  Copyright (c) 2020 FightPandemics
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class SubHeaderOfferRequestCard: UIStackView {
+    let userInitials = UILabel()
+    let userName = UILabel()
+    let userLocationName = UILabel()
+    let userLocationStackView = UIStackView()
+    let userNameAndLocationStackView = UIStackView()
+
+    init(initials: String, name: String, location: String) {
+        super.init(frame: .zero)
+
+        // User Initials
+        let userInitialsSize = CGSize(width: 100, height: 500)
+        userInitials.frame.size = userInitialsSize
+        userInitials.layer.cornerRadius = frame.size.height / 2
+        userInitials.text = initials
+        userInitials.textAlignment = .center
+        userInitials.backgroundColor = .fightPandemicsGhostWhite()
+        userInitials.layer.borderWidth = 1
+        userInitials.layer.borderColor = UIColor.fightPandemicsNeonBlue().cgColor
+        userInitials.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
this also takes the user image, if it has an image, otherwise it takes initials.
also its circle which you can make by giving a corner radius of half the height of the frame.

Reply:
Ok @ethanswift . I need your help on this, please. Tomorrow I will contact you...

Reply:
Ok. I added the View Button but It need layout configuration yet.